app/javascript/modules/listings/DirectoryHelpers.tsx

```diff
@@ -71,15 +71,21 @@ const isNumber = (val: number) => val || val === 0
 // Transforms two numbers into a range string with optional currency formatting and optional suffix
 // 100, 200, true, null --> $100 - $200
 // 100, 100, false, "%" --> 100%
```

Comment:
With the new `forceZeroInRange` argument introduced, existing comments should be updated.